docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-json-api-convention-integration-guide.md

```diff
@@ -0,0 +1,502 @@
+
+class DummyResource extends AbstractResourcePlugin implements JsonApiResourceInterface
+{
+    /**
+     * @return string
+     */
+    public function getType(): string
+    {
+        return 'dummyResourceType';
+    }
+
+    /**
+     * @uses \Spryker\Glue\DummyModuleApi\Controller\ResourceController
+     *
+     * @return string
+     */
+    public function getController(): string
+    {
+        return ResourceController::class;
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\GlueResourceMethodCollectionTransfer
+     */
+    public function getDeclaredMethods(): GlueResourceMethodCollectionTransfer
+    {
+        return (new GlueResourceMethodCollectionTransfer())
+            ->setGetCollection(new GlueResourceMethodConfigurationTransfer())
+            ->setGet(new GlueResourceMethodConfigurationTransfer());
+    }
+}
+```
+
+**src/Pyz/Glue/DummyModuleApi/Plugin/DummyRouteProviderPlugin.php**
+```php
+<?php
+
+namespace Spryker\Glue\DummyModuleApi\Plugin;
+
+use Spryker\Glue\GlueStorefrontApiApplication\Plugin\GlueStorefrontApiApplication\RouteProvider\AbstractRouteProviderPlugin;
+use Spryker\Glue\GlueStorefrontApiApplication\Router\Route\RouteCollection as StorefrontRouteCollection;
+use Spryker\Glue\GlueStorefrontApiApplicationExtension\Dependency\Plugin\RouteProviderPluginInterface as StorefrontRouteProviderPluginInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class DummyRouteProviderPlugin extends AbstractRouteProviderPlugin implements StorefrontRouteProviderPluginInterface
+{
+    /**
+     * @param \Spryker\Glue\GlueStorefrontApiApplication\Router\Route\RouteCollection $routeCollection
+     *
+     * @return \Spryker\Glue\GlueStorefrontApiApplication\Router\Route\RouteCollection
+     */
+    public function addRoutes(StorefrontRouteCollection $routeCollection): StorefrontRouteCollection
+    {
+        $collectionRoute = $this->createGetCollectionRoute('/dummy', 'dummyResourceType');
+        $routeCollection->add('storesCollection', $collectionRoute);
+
+        $route = $this->createGetRoute('/dummy/{id}', 'dummyResourceType');
+        $routeCollection->add('stores', $route);
+
+        return $routeCollection;
+    }
+}
+```
+
+**src/Pyz/Glue/DummyModuleApi/Plugin/DummyResourceRelationshipPlugin.php**
+```php
+<?php
+
+namespace Spryker\Glue\DummyModuleApi\Plugin;
+
+use Generated\Shared\Transfer\GlueRelationshipTransfer;
+use Generated\Shared\Transfer\GlueRequestTransfer;
+use Generated\Shared\Transfer\GlueResourceTransfer;
+use Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\ResourceRelationshipPluginInterface;
+use Spryker\Glue\Kernel\AbstractPlugin;
+
+class DummyCountriesResourceRelationshipPlugin extends AbstractPlugin implements ResourceRelationshipPluginInterface
+{
+    /**
+     * @var string
+     */
+    protected const RESOURCE_TYPE_DUMMY_RELATIONSHIPS = 'dummyRelationshipsType';
+
+    /**
+     * @api
```

Comment:
Clean up the `@api` and `@inheritDoc` - they are irrelevant for the example plugins.